From fa6724770ddb4a98b10527925241aea481c2fd19 Mon Sep 17 00:00:00 2001
From: Philip Chimento <philip.chimento@gmail.com>
Date: Sat, 23 Nov 2019 16:32:31 -0800
Subject: [PATCH] build: Mark SweepingAPI.h as a system header

This is the SpiderMonkey header file that's causing all the remaining
compiler warnings; add it to this file so that it gets marked as a
system header and therefore does not generate warnings.
---
 gjs/jsapi-wrapper.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gjs/jsapi-wrapper.h b/gjs/jsapi-wrapper.h
index fb17b710..e4976200 100644
--- a/gjs/jsapi-wrapper.h
+++ b/gjs/jsapi-wrapper.h
@@ -42,6 +42,7 @@
 // marked as system headers.
 // IWYU pragma: begin_exports
 #include <js/Conversions.h>
+#include <js/SweepingAPI.h>
 #include <jsapi.h>
 #include <jsfriendapi.h>
 // IWYU pragma: end_exports
-- 
2.22.0

